descartes_moveit/src/jaco3_moveit_state_adapter.cpp

```diff
@@ -215,21 +215,21 @@ bool descartes_moveit::Jaco3MoveitStateAdapter::isValid(const std::vector<double
   return !hasNaN(joint_pose) && descartes_moveit::MoveitStateAdapter::isValid(joint_pose);
 }
 
-bool descartes_moveit::Jaco3MoveitStateAdapter::updatePlanningScene(const moveit_msgs::PlanningScene &scene){
-    // Initialize planning scene
-    ROS_INFO("Updating descrates planning scene");
-    planning_scene_->setPlanningSceneMsg(scene);
-    
-    // Update ACM
-    acm_ = planning_scene_->getAllowedCollisionMatrix();
-    // Disable all collision checking
-    acm_.setEntry(true);
-    // Collision check selected arm links with selected robot links
-    acm_.setEntry(collision_robot_links_, collision_arm_links_, false);
-
-    // Initialize collision request message. 
-    // Setting this to false could descrease collision checking speed
-    collision_request_.contacts = true;
+bool descartes_moveit::Jaco3MoveitStateAdapter::updatePlanningScene(planning_scene::PlanningScenePtr ps){
+  // Initialize planning scene
+  planning_scene_ = ps;
+  
+  // Update ACM
+  acm_ = planning_scene_->getAllowedCollisionMatrix();
+  // Disable all collision checking
+  acm_.setEntry(true);
+  // Collision check selected arm links with selected robot links
+  acm_.setEntry(collision_robot_links_, collision_arm_links_, false);
```

Comment:
Is this ACM local or does it affect the ACM for say freespace planning as well?

Reply:
This just a local acm that's used for collision checking by descartes